src/openforms/formio/tests/validation/test_map.py

```diff
@@ -47,5 +127,271 @@ def test_map_field_min_max_length_of_items(self):
 
                 self.assertFalse(is_valid)
                 self.assertIn(component["key"], errors)
```

Comment:
```suggestion
                self.assertIn("foo", errors)
```
I don't like this indirection in tests :) there's only a single component used in all the cases - it'd be a different story if you're looping over components with different keys.